[2024-07-02T15:18:53.756+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: email_teste.tsk4 manual__2024-07-02T15:18:48.647260+00:00 [queued]>
[2024-07-02T15:18:53.763+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: email_teste.tsk4 manual__2024-07-02T15:18:48.647260+00:00 [queued]>
[2024-07-02T15:18:53.763+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-07-02T15:18:53.763+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-07-02T15:18:53.763+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-07-02T15:18:53.773+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): tsk4> on 2024-07-02 15:18:48.647260+00:00
[2024-07-02T15:18:53.777+0000] {standard_task_runner.py:55} INFO - Started process 109 to run task
[2024-07-02T15:18:53.779+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'email_teste', 'tsk4', 'manual__2024-07-02T15:18:48.647260+00:00', '--job-id', '386', '--raw', '--subdir', 'DAGS_FOLDER/email.py', '--cfg-path', '/tmp/tmpvrf7gx87']
[2024-07-02T15:18:53.779+0000] {standard_task_runner.py:83} INFO - Job 386: Subtask tsk4
[2024-07-02T15:18:53.817+0000] {task_command.py:388} INFO - Running <TaskInstance: email_teste.tsk4 manual__2024-07-02T15:18:48.647260+00:00 [running]> on host 47ea7c264468
[2024-07-02T15:18:53.867+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=testeskleuber@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=email_teste
AIRFLOW_CTX_TASK_ID=tsk4
AIRFLOW_CTX_EXECUTION_DATE=2024-07-02T15:18:48.647260+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-07-02T15:18:48.647260+00:00
[2024-07-02T15:18:53.868+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-07-02T15:18:53.868+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'exit 1']
[2024-07-02T15:18:53.879+0000] {subprocess.py:86} INFO - Output:
[2024-07-02T15:18:53.880+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2024-07-02T15:18:53.889+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-07-02T15:18:53.891+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=email_teste, task_id=tsk4, execution_date=20240702T151848, start_date=20240702T151853, end_date=20240702T151853
[2024-07-02T15:18:53.902+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/email.py:152: RemovedInAirflow3Warning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
  send_mime_email(e_from=mail_from, e_to=recipients, mime_msg=msg, conn_id=conn_id, dryrun=dryrun)

[2024-07-02T15:18:53.902+0000] {email.py:268} INFO - Email alerting: attempt 1
[2024-07-02T15:18:56.079+0000] {email.py:280} INFO - Sent an alert email to ['testeskleuber@gmail.com']
[2024-07-02T15:18:58.581+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 386 for task tsk4 (Bash command failed. The command returned a non-zero exit code 1.; 109)
[2024-07-02T15:18:58.604+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-07-02T15:18:58.628+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
